auth/src/main/java/io/grpc/auth/ClientAuthInterceptor.java

```diff
@@ -34,8 +34,8 @@
 import com.google.auth.Credentials;
 import com.google.common.base.Preconditions;
 
+import io.grpc.CallFactory;
```

Comment:
Should it be ClientCallFactory? I think it would need to be for consistency.

The same may apply to unchanged CallOptions?